README.md

```diff
@@ -1,2 +1,44 @@
 # Api-Server
-Lab: 06
+# LAB - 06
+
+## Deployment Test
+
+### Author: Spencer Lazzar
+
+- [tests report](https://github.com/codefellows/code-401-javascript-example-lab/actions)
+- [front-end](https://code-401-js-lab-example.herokuapp.com/status)
+
+### Setup
+
+#### `.env` requirements
+
+- `PORT` - Port Number
+
+#### Running the app
+
+- `npm start`
+- Endpoint: `/status`
+  - Returns Object
+
+    ```javascript
+    {
+      "domain": "john-api-server.demo.herokuapp.com",
+      "status": "running",
+      "port": 42123
+    }
+    ```
+
+#### Tests
+
+- Unit Tests: `npm run test`
+- Lint Tests: `npm run lint`
+
+#### UML
+
+(Created with [diagrams](https://app.diagrams.net/))
+
+![UML Diagram](uml.png)
```

Comment:
This isn't linking properly, if it's not a real link I would like to see an actual UML added here